usaspending_api/accounts/tests/test_federal_account_fy_snapshot_v2.py

```diff
@@ -0,0 +1,86 @@
+import pytest
```

Comment:
If we're including tests in this PR, can we also account for `spending_over_time`, `spending_by_award`, and `spending_by_award_count`?